packages/ozone/src/mod-service/index.ts

```diff
@@ -253,6 +257,11 @@ export class ModerationService {
 
     const meta: Record<string, string | boolean> = {}
 
+    if (isModEventTag(event)) {
+      if (event.add.length) meta.addedTags = event.add.join(' ')
+      if (event.remove.length) meta.removedTags = event.remove.join(' ')
+    }
```

Comment:
Is there a reason these are stuffed in `meta` rather than having their own jsonb columns (i.e. in line with labels)?

Reply:
Another thing that's a little funky here is that the fields `addedTags` and `removedTags` are not reflected in the model.  Even if they end-up in `meta`, we have access to the full range of jsonb, so I'd be all for modeling them as an array rather than a space-separated strings.

Reply:
no reason in particular other than trying to keep the column count low. we would need 2 columns if we want to go with the label approach and for most rows these columns will just be nulls.